atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableMetadata.java

```diff
@@ -294,7 +295,7 @@ public int hashCode() {
         result = prime * result + (negativeLookups? 0 : 1);
         result = prime * result + (sweepStrategy.hashCode());
         result = prime * result + (appendHeavyAndReadLight? 0 : 1);
-        result = prime * result + (nameLoggable? 0 : 1);
+        result = prime * result + nameLogSafety.hashCode(); // Nonnull, because it has a default value
```

Comment:
Line 296 has unnecessary brackets surrounding the `hashCode`. We should probably be consistent - would you like to remove the brackets on 296?